scripts/controllers/main.js

```diff
@@ -1742,30 +1723,51 @@ App.controller('Main', function ($scope, $timeout, $location, Api) {
    };
 
    $scope.openPopupHistory = function (item, entity) {
-      const key = '_popupHistory';
-
-      if (!item[key]) {
-         item[key] = {
-            styles: {
-               width: '100vw',
-               height: '56vw',
-               margin: 0,
-               maxWidth: '100%',
-            },
+      return initPopupLayout(item, entity, '_popupHistory',
+         {
+            classes: ['-popup-landscape'].concat(
+               getItemFieldValue('history.classes', item, entity) || [],
+            ),
```

Comment:
We can use some modern JS features now like destructuring:
```suggestion
            classes: ['-popup-landscape', ...(getItemFieldValue('history.classes', item, entity) || [])],
```
Shorter and cleaner.

Reply:
nice trick. done